mmtk/src/api.rs

```diff
@@ -46,7 +46,9 @@ pub extern "C" fn openjdk_gc_init(calls: *const OpenJDK_Upcalls, heap_size: usiz
         unsafe { &mut *(&*SINGLETON as *const MMTK<OpenJDK> as *mut MMTK<OpenJDK>) };
     memory_manager::gc_init(singleton_mut, heap_size);
     if let PlanSelector::GenCopy = SINGLETON.options.plan {
-        assert!(VMObjectModel::HAS_GC_BYTE, "Side GC-byte conflicts with logging bit metadata.");
+        if cfg!(feature = "side_gc_header") {
```

Comment:
This seems suggesting `side_gc_header` won't work for GenCopy. Is this only for the OpenJDK binding? Theoratically if we have GenCopy in other binding, does it work?